### ISSUE TITLE
implicitly converts incoming `null` to `"NA"` on GeoJSON Features

### ISSUE THREAD

**Issue by onlyjsmith** [disarm-platform/fn-covariate-extractor#18]
We pass in:
```js
...
	properties: {
		n_trials: null
	}
```

and after adding e.g. _elev_, we're getting back:

```js
...
	properties: {
		elev_m: 600,
		n_trials: "NA"
	}
...

**Comment by TNgidi** [comment]
Running the function with the `"NA"` values yields: 
```js
{	"function_status": "error", 
	"result": "Error from function: Series can only be of one type, or None."
}
```

**Comment by onlyjsmith** [comment]
From @HughSt on Twist:

> Hmm, not sure why this is suddenly causing us issues. I've been wanting to switch the UI (ntd-shiny-points) over to the upgraded (and R version) function anyway (fn-prevalence-predictor-mgcv) but last time I ran it, the deployed version was incredibly slow compared to running on my machine. It doesn't look like this function is deployed on the new server (I get a 404).

> Do you know where the function is producing the error? I see that on line 83 of handler.py there is the line
> `train_data = input_data.dropna(axis=0)`
> Which suggests that the function is expecting NAs.

> I'm hesitant to change fn-covariate-extractor as it will have knock on effects on other things that rely on it.

**Comment by onlyjsmith** [comment]
Agree with not wanting to upset things relying on fn-cov-extr, though if it's not returning what it should then we should think hard about adding new code to work around this bug in other functions!

**Comment by onlyjsmith** [comment]
Fixed in v0.2.5
